### DISCUSSION TITLE
Need help to install

### DISCUSSION THREAD

**Discussion by Philipock** [the-marenga/sf-scrapbook-helper#72] [General]
Im new to gitthub and don't know how to install this client is there any way that someone can show me how to install this?

**Comment by the-marenga** [comment]
1. You go on the release page: https://github.com/the-marenga/sf-scrapbook-helper/releases/latest
2. There you click `Windows Download`, or pick the archive for you operating system, if you use another
3. You should then have a zip file in your download folder. You can right click that zip file and "extract all"
    3.5. Alternatively you open the zip via double click and drag&drop the file inside to another folder/desktop
4. This should leave you with a new file "sf-scrapbook-helper.exe", that you can just run via double clicking
